### PR TITLE
Alert

### PR DESCRIPTION
**Pull request by ckahub**
#Alert 
created alert with simple aproach , outline, and icons

### CODE REVIEW

**Comment by som749898** [comment]
looks good

File: Components/Alert/alert.html [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+
+    <title>Alert Component</title>
+        <link rel="stylesheet" href="./alert.css" />
+        <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.4.0/styles/base16/ocean.min.css">
+</head>
+<body>
+
+    <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/11.4.0/highlight.min.js"></script>
+    <script>hljs.highlightAll();</script>
```
Comment (Naveenchandar) [inline Components/Alert/alert.html:16]: Please move scripts either down to body or use async/defer.

File: styles/navBar.css [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    font-family: 'Supermercado One', cursive;
+}
+
+.left_Nav{
+    font-size: 1.2rem;
+    font-weight: 900;
+}
+
+.left_Nav a{
+    color : rgb(48, 46, 46);
```
Comment (Naveenchandar) [inline styles/navBar.css]: Please use hex codes for consistency.

File: styles/sideBar.css [hunk 3]
```diff
@@ -0,0 +1,18 @@
+@import url('https://fonts.googleapis.com/css2?family=Supermercado+One&display=swap');
+
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
```
Comment (Naveenchandar) [inline styles/sideBar.css]: Its repeated in style files. You can move these to utility file.

File: Components/Alert/alert.css [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+}
+
+.warning{
+    background-color: var(--warning-color);
+}
+
+/* 
+<!---outline alert---> */
+
+.outline_Primary{
```
Comment (manujagobind) [inline Components/Alert/alert.css:35]: I'm not sure if this is the right case to follow.

File: Components/Alert/alert.html [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+                      &lt;div class="alert outline_Error"&gt;
+                         &lt;strong&gt;Error!&lt;/strong&gt; This is an Error Alert! check it out.
+                      &lt;/div&gt;
+                    &lt;div class="alert outline_success"&gt;
+                         &lt;strong&gt;Success!&lt;/strong&gt;This is an Success Alert! check it out. 
+                     &lt;/div&gt;
+                     &lt;div class="alert outline_warning"&gt;
+                         &lt;strong&gt;Warning!&lt;/strong&gt;This is an Warning Alert! check it out.
+                     &lt;/div&gt;
+                    </code></pre>
```
Comment (manujagobind) [inline Components/Alert/alert.html:145]: Use prettier to format all HTML

File: styles/navBar.css [hunk 6]
```diff
@@ -0,0 +1,68 @@
+@import url('https://fonts.googleapis.com/css2?family=Supermercado+One&display=swap');
+
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+ 
+}
+
+
```
Comment (manujagobind) [inline styles/navBar.css:10]: Remove extra line breaks

File: styles/navBar.css [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+.small_Text{
+    color :rgb(90, 90, 90);
+    font-size: .8rem;
+    font-weight: 400;
+}
+
+.right_Nav{
+    display: flex;
+   margin-left: auto;
```
Comment (manujagobind) [inline styles/navBar.css]: fix indentation / use a formatting tool to format files

File: Components/Alert/alert.html (hunk 1 above)
Comment (ckahub) [reply Components/Alert/alert.html:16]: THANX I WILL DO SO

**Pull request merged**